packages/realm-react/src/useObject.tsx

```diff
@@ -24,7 +24,27 @@ import { cachedObject } from "./cachedObject";
 // TODO: If we depend on a new version of Realm for @realm/react, we can just use Realm.PrimaryKey
 type PrimaryKey = Parameters<typeof Realm.prototype.objectForPrimaryKey>[1];
 
+/**
+ * Generates the `useObject` hook from a given `useRealm` hook.
+ *
+ * @param useRealm - Hook that returns an open Realm instance
+ * @returns useObject - Hook that is used to gain access to a single Realm object from a primary key
+ */
 export function createUseObject(useRealm: () => Realm) {
+  /**
+   * Returns a realm object from a given type and primary key.
+   * The hook will update on any changes to the properties on the returned object
+   * and return null if it either doesn't exists or has been deleted.
+   *
+   * @example
+   * ```
+   * const object = useObject(Object, objectId);
```

Comment:
Just because I was wondering whether there was a string that had accidentally not been quoted
```suggestion
   * const object = useObject(ObjectClass, objectId);
```